### PR TITLE
Address Issue #32

### PR DESCRIPTION
**Pull request by coreyjewett**

### CODE REVIEW

File: bin/cmd.js [hunk 1]
```diff
@@ -427,8 +430,10 @@ function toStamp (s) {
 }
 
 function getKey (x) {
+    if (x === 'NaN') return strftime(KEY_FORMAT, new Date(NaN));
```
Comment (fnogatz) [inline bin/cmd.js]: Hm, this will always return `'time!NaN-NaN-NaN NaN:NaN:NaN'`...? Is this intended?

Comment (coreyjewett) [reply bin/cmd.js]: Yes. Working around the problem where that key ends up persisted in the db by enabling you to directly address that [broken] record. See my comments on Issue #32.

Comment (fnogatz) [reply bin/cmd.js]: Wouldn't it be better to avoid `NaN` records in the first place? :)

Comment (coreyjewett) [reply bin/cmd.js]: Certainly. However, I already had a corrupted record with the NaN stamp. Until a complete fix is worked out, this at least lets the problem be corrected.

**Comment by fnogatz** [comment]
Thank you for this pull request! Sorry for taking so long to make it into the package... :/

**Comment by coreyjewett** [comment]
no worries. Thanks.

**Pull request merged**